sbndqm/dqmAnalysis/PMT/ICARUS/fcl/icarus_beam_timing_monitor.fcl

```diff
@@ -0,0 +1,54 @@
+#include "redis_connection.fcl"
+#include "icarus_online_decoder_defs.fcl"
+#include "icarus_pmt_analysis_defs.fcl"
+
+process_name: ICARUSBEAMTIMING
+
+physics: {
+  
+  producers: {
+
+    daqPMT: @local::decodePMTOnline
+    daqTrigger: @local::decodeTriggerOnline
+
+  }
+
+  analyzers: {
+
+   timingana: @local::timingAnalysis
+    
+  }
+
+  my_producer_modules: [ daqTrigger, daqPMT ]
+  trigger_paths: [ my_producer_modules ]
+  
+  ana: [ timingana ] 
+  end_paths: [ ana ]
+
+}
+
+services: {
+
+  TFileService: { fileName: "waveformtest.root"}
+
+  RedisConnection: @local::redis_connection
+
+  InitMetricManager: {
+    connections: {
+      fast: @local::redis_metric_fast
+      slow: @local::redis_metric_slow
+      archive: @local::redis_metric_archive
+    }
+  }
+
+} 
+
+
+services.RedisConnection.host: "icarus-db01.fnal.gov"
```

Comment:
Isn't this the same URL that was changed to `db02` in another FHiCL configuration of this PR?

Reply:
That's right. Since `icarus-db01` is dead (RIP), I changed the host in the primary fcls (under `installation\sbn-fd`) but forgot to change it in the various "standalone" analysis. I'll fix it as well.